### ISSUE TITLE
Doesn't work with GL73

### ISSUE THREAD

**Issue by Douglas-Hsieh** [Askannz/msi-perkeyrgb#32]
I'm on pop os.
I've tried:
$ msi-perkeyrgb -m <model> -s 00ff00
for each of the models. The command finishes with no output and my keyboard doesn't change.

**Comment by erikbdev** [comment]
Does your variant keyboard support per-key-rgb? If it's just a single color then unfortunately it will not work.

**Comment by Douglas-Hsieh** [comment]
Yeah, on Windows the default setting is a flowing rainbow lighting.

**Comment by bwu62** [comment]
I just installed this on my GL73 (specifically GL73 9SDK-413) and it works no problem. I've tried the default mode (GE63) as well as a few others (like GE73 and GS73) and they all seem to work.

I think you should be able to add GL73 to the list of officially supported devices.
